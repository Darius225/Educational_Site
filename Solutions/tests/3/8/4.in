3000
22 34 49 51 55 55 60 62 70 95 106 111 113 117 123 123 131 172 176 263 277 278 292 294 308 309 316 328 332 377 404 407 409 414 422 427 437 440 464 478 491 495 497 505 510 527 536 540 568 574 585 591 594 610 616 616 651 687 687 692 692 695 703 714 717 757 757 780 792 795 797 833 835 860 870 895 896 937 942 951 952 967 971 984 996 1008 1014 1022 1035 1037 1051 1056 1073 1083 1090 1112 1125 1139 1140 1158 1161 1163 1168 1172 1180 1197 1197 1214 1217 1221 1238 1252 1307 1311 1342 1349 1358 1413 1449 1462 1490 1509 1510 1511 1513 1517 1550 1552 1563 1579 1584 1588 1589 1607 1608 1619 1634 1640 1684 1701 1704 1715 1722 1723 1724 1733 1747 1763 1768 1776 1779 1807 1813 1813 1826 1843 1889 1897 1922 1924 1933 1952 1965 1980 1997 2017 2029 2029 2036 2039 2051 2052 2058 2059 2084 2090 2095 2100 2109 2111 2111 2114 2122 2129 2130 2175 2193 2217 2218 2230 2230 2232 2236 2238 2245 2247 2266 2266 2272 2275 2284 2339 2386 2396 2403 2403 2440 2446 2448 2454 2490 2505 2512 2519 2522 2522 2524 2554 2571 2595 2604 2606 2628 2647 2648 2659 2664 2668 2671 2689 2701 2704 2711 2712 2717 2718 2727 2735 2736 2742 2748 2775 2777 2778 2780 2783 2784 2798 2803 2821 2828 2874 2892 2903 2909 2920 2926 2928 2929 2939 2949 2953 2963 2969 2997 3001 3002 3017 3038 3050 3050 3053 3058 3071 3077 3090 3090 3109 3128 3130 3142 3154 3164 3169 3173 3193 3204 3225 3245 3267 3278 3285 3303 3303 3324 3333 3336 3377 3382 3390 3413 3421 3421 3437 3465 3474 3478 3485 3500 3509 3524 3536 3565 3569 3575 3575 3613 3614 3620 3625 3632 3635 3637 3637 3641 3649 3657 3666 3684 3695 3711 3725 3735 3742 3753 3756 3780 3790 3792 3820 3863 3869 3884 3893 3895 3899 3900 3903 3905 3906 3907 3914 3947 3959 3962 3972 3982 3994 3999 4014 4018 4019 4032 4039 4046 4087 4091 4096 4099 4100 4104 4107 4111 4142 4144 4145 4146 4169 4180 4202 4237 4240 4296 4322 4347 4348 4361 4364 4382 4390 4414 4416 4418 4427 4431 4433 4443 4445 4451 4468 4470 4470 4472 4479 4489 4491 4527 4536 4536 4543 4546 4546 4558 4559 4581 4585 4598 4620 4622 4662 4672 4679 4687 4689 4698 4705 4714 4729 4748 4768 4780 4780 4784 4787 4791 4799 4799 4802 4803 4816 4822 4871 4888 4906 4907 4909 4948 4955 4989 5001 5007 5008 5037 5073 5076 5088 5121 5143 5153 5162 5162 5164 5170 5186 5197 5200 5220 5221 5227 5228 5249 5253 5261 5269 5269 5303 5306 5311 5319 5332 5339 5346 5379 5380 5394 5398 5459 5485 5490 5504 5507 5508 5520 5524 5528 5555 5555 5556 5560 5570 5574 5613 5615 5618 5643 5675 5683 5684 5695 5705 5712 5714 5728 5742 5745 5758 5764 5765 5811 5834 5845 5849 5855 5887 5891 5905 5918 5924 5928 5936 5958 5961 5965 5990 5999 6017 6023 6026 6028 6030 6031 6042 6070 6094 6095 6096 6113 6121 6133 6143 6149 6161 6167 6177 6185 6189 6189 6204 6208 6217 6224 6226 6262 6291 6296 6304 6315 6324 6336 6338 6344 6357 6357 6372 6373 6382 6398 6407 6411 6421 6421 6470 6475 6480 6487 6489 6492 6527 6530 6543 6547 6570 6597 6605 6638 6641 6648 6679 6680 6686 6705 6713 6720 6736 6755 6775 6791 6798 6810 6819 6826 6834 6869 6889 6905 6908 6913 6916 6947 6949 6960 6964 6967 6972 6981 6998 7008 7009 7011 7014 7015 7018 7056 7067 7082 7098 7100 7105 7116 7121 7138 7146 7146 7149 7151 7176 7184 7189 7195 7196 7201 7202 7226 7244 7245 7252 7254 7263 7275 7277 7300 7339 7348 7354 7376 7386 7408 7430 7441 7442 7454 7456 7476 7483 7485 7496 7497 7520 7589 7600 7607 7616 7619 7621 7636 7640 7654 7659 7672 7706 7718 7723 7748 7756 7763 7794 7804 7805 7810 7825 7826 7836 7845 7860 7871 7881 7885 7896 7897 7913 7916 7918 7918 7922 7927 7928 7934 7950 7951 7965 7981 7999 8015 8018 8023 8035 8041 8043 8044 8062 8072 8076 8091 8105 8119 8122 8126 8152 8163 8164 8177 8179 8182 8249 8257 8264 8267 8270 8280 8292 8298 8298 8314 8323 8329 8350 8370 8375 8380 8383 8387 8400 8408 8409 8426 8429 8434 8448 8448 8449 8454 8466 8482 8489 8493 8511 8514 8522 8533 8546 8547 8568 8589 8610 8625 8628 8632 8632 8636 8642 8676 8677 8690 8691 8719 8729 8746 8749 8749 8775 8792 8796 8810 8832 8835 8837 8848 8866 8866 8871 8884 8899 8917 8958 9000 9003 9006 9013 9017 9019 9021 9029 9031 9034 9045 9051 9074 9084 9105 9106 9143 9168 9178 9178 9180 9183 9185 9190 9196 9222 9224 9235 9241 9259 9265 9270 9286 9290 9309 9309 9319 9343 9350 9363 9392 9424 9424 9443 9456 9466 9468 9483 9496 9525 9528 9555 9558 9560 9568 9571 9572 9578 9579 9598 9604 9639 9662 9668 9676 9679 9682 9705 9707 9730 9753 9757 9760 9761 9767 9771 9773 9778 9784 9784 9798 9799 9801 9813 9833 9848 9853 9878 9882 9896 9898 9908 9911 9913 9913 9926 9931 9949 9950 9950 9953 9976 9986 9986 10017 10058 10061 10072 10080 10088 10100 10108 10109 10115 10116 10121 10132 10136 10151 10155 10157 10162 10169 10188 10188 10198 10198 10205 10205 10227 10238 10240 10248 10264 10266 10267 10297 10322 10324 10336 10342 10348 10360 10360 10391 10403 10404 10409 10424 10431 10449 10449 10474 10487 10508 10508 10528 10534 10543 10556 10560 10570 10576 10580 10583 10588 10591 10595 10612 10627 10631 10634 10659 10668 10669 10677 10687 10697 10713 10714 10721 10740 10778 10784 10790 10794 10799 10803 10804 10815 10822 10828 10844 10865 10872 10896 10908 10917 10930 10948 10954 10963 10963 10966 10986 10996 11005 11024 11047 11056 11065 11070 11111 11130 11167 11168 11210 11217 11246 11268 11282 11291 11314 11315 11331 11338 11358 11364 11367 11382 11413 11416 11419 11423 11427 11429 11436 11436 11446 11481 11485 11490 11491 11537 11541 11552 11554 11562 11566 11581 11586 11606 11607 11628 11629 11639 11647 11649 11655 11667 11670 11678 11700 11706 11709 11726 11727 11735 11735 11740 11740 11755 11770 11774 11783 11788 11801 11806 11824 11830 11867 11883 11899 11904 11916 11917 11918 11932 11947 11976 11998 12008 12017 12030 12036 12050 12085 12100 12120 12122 12132 12135 12144 12148 12155 12157 12171 12176 12188 12203 12211 12215 12249 12269 12287 12293 12294 12301 12320 12325 12327 12367 12376 12401 12422 12432 12439 12452 12453 12481 12492 12499 12531 12531 12581 12598 12602 12622 12635 12653 12661 12687 12728 12738 12740 12755 12756 12756 12793 12796 12796 12800 12807 12808 12816 12827 12834 12903 12921 12922 12944 12950 12957 12961 12966 12981 12986 13005 13009 13016 13020 13022 13051 13059 13063 13065 13068 13089 13102 13137 13188 13211 13213 13222 13222 13224 13229 13235 13249 13252 13255 13264 13266 13270 13274 13279 13289 13299 13304 13315 13334 13350 13370 13374 13375 13376 13380 13401 13413 13413 13424 13441 13465 13467 13525 13532 13535 13538 13543 13572 13585 13589 13591 13601 13607 13608 13610 13611 13612 13692 13725 13726 13735 13745 13753 13771 13776 13790 13792 13811 13819 13829 13845 13847 13848 13850 13862 13890 13890 13894 13894 13901 13904 13912 13913 13936 13971 13989 13998 14011 14029 14034 14035 14044 14048 14077 14090 14097 14099 14102 14112 14117 14133 14135 14147 14178 14187 14204 14214 14224 14227 14245 14271 14278 14287 14301 14308 14320 14322 14362 14365 14378 14392 14396 14398 14402 14404 14411 14418 14422 14439 14452 14470 14475 14498 14499 14511 14522 14526 14542 14553 14557 14578 14578 14610 14616 14617 14639 14652 14654 14655 14692 14707 14734 14761 14764 14791 14815 14816 14819 14819 14823 14841 14852 14857 14857 14869 14871 14882 14894 14898 14946 14958 14966 14986 15000 15002 15041 15045 15050 15056 15074 15075 15114 15138 15149 15163 15167 15177 15196 15211 15216 15229 15248 15255 15267 15279 15279 15299 15305 15310 15335 15337 15346 15347 15369 15390 15411 15425 15431 15439 15443 15452 15467 15476 15493 15510 15518 15520 15532 15533 15543 15555 15557 15559 15568 15577 15577 15587 15601 15604 15606 15609 15619 15630 15641 15650 15658 15683 15739 15765 15769 15777 15800 15825 15834 15841 15850 15865 15878 15902 15923 15932 15937 15937 15939 15947 15956 15963 15966 15974 15991 15992 15996 16014 16020 16070 16077 16099 16102 16122 16143 16147 16158 16177 16186 16209 16221 16227 16232 16246 16269 16272 16281 16288 16293 16302 16318 16318 16336 16337 16342 16344 16344 16351 16356 16413 16415 16427 16477 16481 16487 16490 16492 16503 16506 16511 16515 16529 16550 16554 16570 16574 16577 16604 16607 16612 16622 16646 16658 16666 16672 16673 16677 16688 16699 16704 16706 16709 16727 16728 16739 16745 16775 16780 16787 16796 16797 16801 16827 16833 16846 16851 16859 16863 16870 16872 16877 16880 16882 16895 16904 16914 16915 16930 16937 16971 16994 16997 17033 17048 17076 17100 17103 17118 17125 17140 17146 17148 17150 17164 17165 17189 17191 17207 17217 17221 17238 17240 17241 17271 17275 17282 17295 17335 17338 17341 17348 17356 17357 17358 17362 17369 17372 17389 17389 17400 17400 17401 17426 17433 17447 17469 17487 17496 17497 17530 17530 17543 17544 17544 17554 17594 17594 17606 17618 17633 17648 17652 17655 17657 17661 17664 17675 17685 17691 17698 17699 17702 17708 17708 17713 17723 17745 17747 17771 17789 17789 17801 17868 17872 17875 17882 17900 17912 17918 17922 17923 17926 17929 17938 17962 17981 17986 17996 18002 18008 18009 18012 18019 18045 18050 18051 18054 18055 18093 18105 18114 18129 18149 18170 18171 18173 18178 18217 18223 18228 18239 18248 18254 18255 18286 18294 18294 18302 18309 18315 18321 18335 18389 18402 18412 18414 18414 18444 18474 18483 18503 18510 18525 18559 18562 18567 18572 18576 18576 18586 18595 18598 18599 18607 18622 18625 18648 18653 18665 18699 18703 18710 18725 18755 18769 18774 18809 18818 18825 18837 18852 18889 18905 18907 18921 18927 18960 18963 18965 18979 19042 19045 19050 19062 19081 19087 19095 19104 19120 19126 19126 19127 19129 19139 19147 19160 19163 19180 19183 19186 19190 19199 19201 19203 19203 19205 19207 19207 19214 19223 19228 19232 19238 19253 19266 19273 19283 19305 19306 19307 19324 19336 19366 19377 19402 19415 19429 19435 19479 19501 19509 19521 19522 19522 19533 19537 19537 19564 19564 19572 19592 19622 19636 19642 19652 19653 19658 19662 19662 19668 19683 19686 19695 19696 19713 19741 19765 19769 19773 19789 19793 19819 19823 19839 19847 19854 19869 19884 19886 19888 19896 19901 19904 19909 19914 19916 19919 19932 19948 19958 19962 19966 19969 19971 19994 20007 20012 20020 20024 20034 20045 20070 20095 20103 20112 20134 20140 20144 20147 20151 20175 20206 20211 20217 20225 20233 20262 20266 20283 20293 20312 20312 20326 20327 20329 20331 20336 20336 20346 20398 20439 20442 20447 20452 20469 20492 20494 20506 20517 20518 20526 20527 20529 20541 20554 20564 20580 20582 20626 20639 20651 20652 20659 20687 20700 20701 20711 20729 20763 20767 20767 20801 20830 20862 20863 20874 20886 20888 20894 20913 20929 20940 20941 20950 20954 21006 21008 21016 21018 21049 21061 21062 21074 21078 21082 21085 21107 21112 21114 21126 21131 21149 21154 21164 21182 21188 21203 21210 21230 21232 21232 21236 21253 21263 21267 21280 21287 21293 21320 21334 21338 21344 21369 21370 21371 21407 21429 21436 21457 21486 21490 21494 21510 21516 21526 21539 21541 21556 21556 21557 21564 21568 21582 21587 21642 21658 21677 21683 21684 21687 21690 21703 21705 21709 21743 21747 21748 21752 21756 21759 21772 21779 21782 21782 21801 21815 21824 21826 21835 21838 21842 21845 21851 21852 21872 21892 21903 21918 21920 21932 21934 21949 21950 21954 21972 21976 21978 22004 22012 22036 22037 22048 22060 22061 22080 22082 22087 22093 22097 22102 22104 22129 22130 22141 22144 22165 22189 22201 22203 22221 22235 22249 22275 22283 22288 22295 22307 22324 22334 22347 22351 22355 22392 22408 22422 22422 22423 22434 22454 22456 22456 22477 22503 22511 22519 22524 22531 22549 22573 22581 22583 22586 22597 22598 22602 22629 22630 22632 22636 22644 22647 22648 22674 22677 22678 22684 22689 22707 22718 22726 22772 22780 22780 22783 22832 22837 22844 22848 22854 22864 22870 22903 22908 22909 22915 22918 22927 22929 22933 22936 22986 23001 23002 23006 23013 23019 23040 23041 23048 23050 23052 23054 23061 23065 23066 23090 23108 23125 23135 23154 23154 23160 23169 23177 23180 23181 23189 23197 23206 23224 23224 23243 23259 23261 23265 23266 23298 23303 23319 23331 23342 23347 23350 23355 23361 23401 23414 23442 23446 23456 23461 23467 23480 23482 23502 23514 23548 23552 23571 23574 23591 23596 23597 23603 23608 23610 23616 23627 23655 23657 23662 23662 23670 23679 23692 23719 23720 23723 23761 23791 23793 23829 23836 23845 23877 23892 23921 23924 23967 23977 23978 23992 23998 24008 24011 24013 24018 24037 24042 24052 24058 24068 24073 24081 24095 24100 24123 24131 24151 24159 24170 24175 24176 24178 24180 24182 24188 24243 24252 24255 24273 24279 24284 24286 24297 24297 24305 24305 24309 24320 24331 24334 24342 24344 24359 24361 24372 24411 24433 24435 24436 24438 24440 24458 24474 24487 24541 24560 24577 24579 24583 24597 24636 24645 24647 24660 24664 24673 24761 24771 24772 24777 24806 24812 24812 24814 24820 24831 24832 24853 24856 24859 24871 24878 24885 24896 24896 24902 24924 24954 24957 24958 24968 24995 25008 25037 25044 25046 25058 25063 25081 25085 25085 25098 25104 25117 25122 25126 25155 25185 25205 25207 25208 25211 25224 25232 25244 25277 25282 25304 25316 25340 25360 25367 25368 25370 25383 25387 25390 25393 25399 25402 25412 25421 25432 25437 25473 25476 25480 25484 25514 25521 25546 25553 25565 25590 25596 25608 25621 25649 25655 25660 25667 25672 25676 25696 25715 25727 25731 25734 25741 25750 25756 25780 25784 25808 25826 25839 25840 25847 25847 25851 25855 25860 25874 25880 25896 25909 25916 25932 25956 25968 25996 26027 26028 26039 26046 26047 26056 26059 26062 26063 26075 26087 26090 26091 26093 26146 26152 26154 26154 26154 26174 26178 26188 26188 26195 26209 26222 26259 26265 26292 26293 26295 26305 26329 26359 26361 26378 26381 26385 26393 26396 26420 26431 26434 26448 26475 26497 26500 26505 26506 26518 26542 26549 26557 26569 26571 26575 26579 26591 26594 26606 26623 26625 26648 26649 26649 26651 26672 26680 26680 26682 26688 26694 26695 26717 26723 26730 26735 26739 26771 26810 26811 26814 26821 26821 26831 26834 26841 26843 26847 26869 26878 26887 26895 26910 26947 26952 26963 26974 26977 26981 26981 26986 27003 27024 27032 27039 27073 27080 27083 27108 27118 27128 27132 27157 27164 27168 27175 27182 27192 27202 27203 27219 27221 27233 27234 27244 27265 27266 27270 27278 27322 27328 27349 27357 27358 27359 27360 27383 27384 27395 27396 27438 27445 27458 27481 27494 27506 27507 27518 27543 27552 27597 27597 27601 27611 27633 27638 27639 27651 27678 27688 27700 27716 27735 27743 27752 27756 27765 27767 27778 27790 27792 27792 27812 27844 27857 27865 27867 27868 27886 27908 27911 27915 27918 27921 27922 27929 27944 27956 27983 28009 28023 28030 28037 28046 28052 28056 28056 28060 28061 28071 28072 28101 28104 28104 28113 28125 28152 28157 28159 28168 28196 28232 28236 28252 28266 28274 28277 28287 28289 28299 28300 28302 28304 28309 28311 28319 28322 28325 28379 28400 28411 28418 28418 28422 28439 28445 28468 28488 28494 28524 28544 28571 28580 28590 28596 28606 28618 28623 28630 28637 28652 28659 28665 28694 28719 28725 28731 28765 28767 28775 28776 28780 28793 28795 28816 28817 28839 28851 28870 28875 28883 28888 28892 28904 28909 28913 28936 28942 28948 28952 28952 28952 28962 28975 28980 28989 28994 29032 29061 29065 29072 29076 29079 29110 29123 29123 29127 29128 29130 29141 29147 29176 29225 29232 29237 29237 29244 29265 29271 29296 29298 29298 29299 29315 29330 29342 29347 29364 29379 29393 29396 29398 29402 29428 29431 29444 29455 29456 29461 29480 29489 29491 29538 29543 29554 29554 29559 29578 29589 29589 29596 29612 29617 29644 29648 29655 29684 29700 29705 29713 29717 29739 29741 29745 29759 29766 29781 29817 29820 29831 29870 29875 29897 29906 29958 29989 29997 30008 30022 30032 30042 30042 30048 30064 30072 30080 30098 30102 30125 30134 30134 30138 30162 30180 30187 30201 30205 30210 30213 30229 30231 30237 30261 30271 30278 30283 30286 30287 30300 30315 30335 30346 30355 30364 30371 30394 30397 30399 30401 30402 30403 30424 30438 30446 30447 30460 30471 30482 30488 30499 30500 30518 30523 30524 30550 30581 30600 30606 30611 30615 30625 30630 30635 30655 30663 30678 30679 30680 30689 30751 30762 30769 30781 30795 30800 30801 30824 30861 30872 30876 30877 30924 30924 30941 30948 30964 30987 30990 30991 30998 31004 31038 31043 31043 31059 31059 31062 31069 31094 31114 31122 31132 31136 31141 31144 31149 31159 31181 31196 31196 31213 31214 31219 31247 31250 31251 31281 31284 31294 31301 31318 31325 31337 31391 31415 31427 31429 31432 31434 31454 31459 31460 31476 31479 31501 31520 31522 31530 31558 31558 31578 31586 31600 31629 31634 31651 31659 31659 31663 31668 31671 31706 31710 31728 31747 31750 31754 31783 31784 31785 31800 31801 31808 31855 31858 31860 31873 31877 31881 31891 31914 31916 31917 31933 31934 31942 31944 31945 31946 31950 31956 31983 31987 31997 32001 32004 32025 32037 32043 32056 32064 32073 32081 32086 32087 32090 32092 32100 32116 32117 32131 32133 32149 32161 32163 32183 32194 32213 32236 32239 32253 32282 32286 32287 32299 32306 32314 32321 32333 32340 32340 32345 32348 32360 32374 32380 32408 32418 32443 32471 32483 32486 32492 32497 32498 32500 32500 32509 32526 32560 32561 32579 32582 32582 32603 32604 32612 32618 32638 32697 32703 32720 32721 32723 32750 
30
997
2115
1013
1775
1900
2808
1813
1571
2874
1467
2098
2205
268
2841
1634
2740
435
2211
1083
342
1713
163
813
1408
728
705
2761
2530
944
87
