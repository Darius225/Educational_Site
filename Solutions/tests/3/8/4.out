-1
-1
-1
-1
-1
-1
154
-1
252
-1
-1
-1
-1
-1
137
-1
-1
-1
94
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
